browser/app/less/inc/alert.less

```diff
@@ -36,33 +31,32 @@
     }
 
     &.alert-info {
-        background: @blue;
+        background: @light-blue;
     }
 
-    @media(max-width: (@screen-xs-max)) {
+    @media (max-width: (@screen-xs-max)) {
         left: 20px;
         width: ~"calc(100% - 40px)";
         max-width: 100%;
     }
 
-    .progress {
-        margin: 10px 10px 8px 0;
-        height: 5px;
-        box-shadow: none;
-        border-radius: 1px;
-        background-color: @blue;
-        border-radius: 2px;
-        overflow: hidden;
-    }
-
-    .progress-bar {
-        box-shadow: none;
-        background-color: @white;
-        height: 100%;
-    }
-
     .close {
```

Comment:
Again from [the old PR](https://github.com/minio/minio/pull/4697#discussion_r128388936). The close icon isn't correctly centered. On firefox 54.0.1.

![screen shot 2017-07-25 at 5 11 37 pm](https://user-images.githubusercontent.com/3038254/28599057-8cb2aed8-715c-11e7-93e8-2b9e54eec978.png)